kiva/fonttools/font_manager.py

```diff
@@ -875,7 +875,7 @@ def get_name(self):
             return afm.AFM(open(filename)).get_familyname()
 
         font = fontManager.findfont(self)
-        return getPropDict(TTFont(six.text_type(font))['name'])
+        return getPropDict(TTFont(six.text_type(font)))['name']
```

Comment:
do you think it will help readability if you change the formatting here to 

```
getPropDict(
    TTFont(six.text_type(font))
)['name']
```

or something of that sort?

Reply:
Maybe even breaking that out of the `return` altogether to make it more readable. Something like:
```
prop_dict = getPropDict(TTFont(six.text_type(font))
return prop_dict['name']
```

What do you think  @rahulporuri ?

Reply:
👍